.github/workflows/import-figma.yml

```diff
@@ -27,22 +27,12 @@ jobs:
       # we need to separately import the themes because they are all needed in different formats and selectors
       - name: 🎨 Import onyx variables
         run: |
-          pnpm run @sit-onyx/figma-utils import-variables -k "${{ vars.FIGMA_FILE_KEY }}" -t "${{ secrets.FIGMA_TOKEN }}" -d "../sit-onyx/src/styles/variables/themes" -m onyx -f CSS -s ":where(:root), .onyx-theme-default"
+          pnpm run @sit-onyx/figma-utils import-variables -k "${{ vars.FIGMA_FILE_KEY }}" -t "${{ secrets.FIGMA_TOKEN }}" -d "../sit-onyx/src/styles/variables/themes" -m Value onyx-light onyx-dark -f CSS -s ":where(:root), .onyx-theme-default"
         working-directory: packages/figma-utils
 
       - name: 🎨 Import additional onyx themes
         run: |
-          pnpm run @sit-onyx/figma-utils import-variables -k "${{ vars.FIGMA_FILE_KEY }}" -t "${{ secrets.FIGMA_TOKEN }}" -d "../sit-onyx/src/styles/variables/themes" -m lidl kaufland twogo -f CSS -s ":where(:root), .onyx-theme-{mode}"
-        working-directory: packages/figma-utils
-
-      - name: ☀️ Import light variables
-        run: |
-          pnpm run @sit-onyx/figma-utils import-variables -k "${{ vars.FIGMA_FILE_KEY }}" -t "${{ secrets.FIGMA_TOKEN }}" -d "../sit-onyx/src/styles/variables" -m light -s ":where(:root)"
-        working-directory: packages/figma-utils
-
-      - name: 🌙 Import dark variables
-        run: |
-          pnpm run @sit-onyx/figma-utils import-variables -k "${{ vars.FIGMA_FILE_KEY }}" -t "${{ secrets.FIGMA_TOKEN }}" -d "../sit-onyx/src/styles/variables" -m dark -s ":where(.dark)"
+          pnpm run @sit-onyx/figma-utils import-variables -k "${{ vars.FIGMA_FILE_KEY }}" -t "${{ secrets.FIGMA_TOKEN }}" -d "../sit-onyx/src/styles/variables/themes" -m lidl-light lidl-dark kaufland-light kaufland-dark digits-light digits-dark -f CSS -s ":where(:root), .onyx-theme-{mode}"
```

Comment:
light and dark themes must be imported separately with a different selector, otherwise both will always be applied with `:where(:root)`.

Dark themes must use the `:where(.dark)` selector instead of `:where(:root)`

Reply:
I changed the imports in the theming PR.